core/src/main/java/com/graphhopper/storage/BaseGraph.java

```diff
@@ -493,6 +493,13 @@ public int getSegmentSize() {
         return segmentSize;
     }
 
+    /**
+     * Currently MMapDataAccess cannot be used when potentially writing to the same DataAccess from multiple threads.
+     */
+    public boolean areEdgesConcurrentWritable() {
+        return store.areEdgesConcurrentWritable();
+    }
```

Comment:
Setting encoded values like `xyz_subnetwork` isn't thread-safe for RAM DataAccess, or am I missing something? Basically we would potentially be modifying the same int from different threads in different ways?